examples/tensorflow/sagemaker_official_container/horovod_estimator_mnist.py

```diff
@@ -0,0 +1,223 @@
+#  Copyright 2018 Uber Technologies, Inc. All Rights Reserved.
+#  Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
```

Comment:
Is there a reason we're including the license here but not in the other horovod examples?

Reply:
Strangely the other example I took didn't have such a license